onnxruntime/contrib_ops/cuda/activation/activations_impl.cu

```diff
@@ -38,8 +38,12 @@ struct OP_ScaledTanh : public CtxScaledTanh {
 
 template <typename T>
 struct OP_Gelu : public CtxGelu {
```

Comment:
This approximation is very close but not exactly the same. Once there are 12 layers, the error could accumulate to be large enough. I observed this in GPT2 model. I suggest to keep both, and use an attribute in operator to choose one.